astroid/node_classes.py

```diff
@@ -1053,15 +1053,17 @@ def postinit(self, targets=None, value=None):
 
 class AnnAssign(mixins.AssignTypeMixin, Statement):
     """class representing an AnnAssign node"""
-    _astroid_fields = ('target', 'value', 'annotation',)
+    _astroid_fields = ('target', 'annotation', 'value', 'simple')
     target = None
-    value = None
     annotation = None
+    value = None
+    simple = None
 
-    def postinit(self, target=None, value=None, annotation=None):
+    def postinit(self, target=None, annotation=None, value=None, simple=None):
```

Comment:
https://docs.python.org/3/library/ast.html
```
          -- 'simple' indicates that we annotate simple name without parens
          | AnnAssign(expr target, expr annotation, expr? value, int simple)
```

Looks like only `value` is optional